instrumentation/trilogy/lib/opentelemetry/instrumentation/trilogy/patches/client.rb

```diff
@@ -68,11 +63,12 @@ def query(sql)
           def client_attributes(sql)
             attributes = {
               ::OpenTelemetry::SemanticConventions::Trace::DB_SYSTEM => 'mysql',
-              ::OpenTelemetry::SemanticConventions::Trace::NET_PEER_NAME => net_peer_name
+              ::OpenTelemetry::SemanticConventions::Trace::NET_PEER_NAME => connection_options.fetch(:host, 'unknown sock'),
```

Comment:
```suggestion
              ::OpenTelemetry::SemanticConventions::Trace::NET_PEER_NAME => connection_options.fetch(:host, 'unknown sock')
```

to make the linter happy